### DISCUSSION TITLE
Rolling auto-pruning of results?

### DISCUSSION THREAD

**Discussion by Jaxel** [alexjustesen/speedtest-tracker#877] [Ideas]
I don't know about you guys, but I don't need the results to my speedtests from 6 months ago. Is there a way to set it up so that result logs older than 28 days automatically get deleted?

**Comment by alexjustesen** [comment]
At the moment no and I think I chatted in the past about adding this as I think it's a good idea. Going to convert this to an issue and tag as a feature so it can be worked on in the future.

**Reply by alexjustesen** [reply]
Dup of https://github.com/alexjustesen/speedtest-tracker/discussions/703, closing
